amazonpay/common_type.go

```diff
@@ -17,9 +17,12 @@ type ProviderMetadata struct {
 }
 
 type Buyer struct {
-	BuyerID string `json:"buyerId,omitempty"`
-	Name    string `json:"name,omitempty"`
-	Email   string `json:"email,omitempty"`
+	BuyerID         string `json:"buyerId,omitempty"`
+	Name            string `json:"name,omitempty"`
+	Email           string `json:"email,omitempty"`
+	PostalCode      string `json:"postalCode,omitempty"`
+	ShippingAddress AddressDetails
+	PhoneNumber     string `json:"phoneNumber,omitempty"`
```

Comment:
"Checkout Session" and "Charge Permission" are used by Buyer. While the phone number can be confirmed, are the postal code and shipping address also defined in the response? If there is any reference documentation available, I would like to know.